README.md

```diff
@@ -47,19 +47,25 @@ More detailed build instructions can be found [here](docs/install_vortex.md).
 - [Yosys](https://github.com/YosysHQ/yosys)
 - [Sv2v](https://github.com/zachjs/sv2v)
 ### Install development tools
-    $ sudo apt-get install build-essential
-    $ sudo apt-get install binutils
-    $ sudo apt-get install python
-    $ sudo apt-get install uuid-dev
-    $ sudo apt-get install git
+    $ sudo apt-get install build-essential git binutils python3 uuid-dev wget ocl-icd-opencl-dev
 ### Install Vortex codebase
-    $ git clone --depth=1 --recursive https://github.com/vortexgpgpu/vortex.git
-    $ cd Vortex
+    $ git clone https://github.com/vortexgpgpu/vortex.git
+    $ cd vortex
+    $ git submodule sync
```

Comment:
Was this one necessary for you after a fresh clone?
```suggestion
```

Reply:
since I got rid of this

```bash
git clone --depth=1 --recursive https://github.com/vortexgpgpu/vortex.git
```


it makes it necessary otherwise ramulator and other third parties are missing